schemas/2019-03-01-hybrid/deploymentTemplate.json

```diff
@@ -0,0 +1,3639 @@
+        },
+        {
+          "properties": {
+            "apiProfile": {
+              "enum": [
+                "2019-03-01-hybrid"
+              ]
+            },
+            "resources": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/2019-03-01-hybrid-profile-resource"
+              }
+            }
+          }
+        },
+        {
+          "properties": {
+            "apiProfile": {
+              "enum": [
+                "2018-06-01-profile"
+              ]
+            },
+            "resources": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/2019-04-01-profile-resource"
+              }
+            }
+          }
+        },
+        {
+          "properties": {
+            "apiProfile": {
+              "enum": [
+                "2017-03-09-profile"
+              ]
+            },
+            "resources": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/2019-04-01-profile-resource"
+              }
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "properties": {
+    "resources": {
+      "type": "array"
+    },
+    "apiProfile": {
+      "type": "string",
+      "enum": [
+        "2018-03-01-hybrid",
+        "2019-03-01-hybrid",
+        "2017-03-09-profile",
+        "2018-06-01-profile"
+      ],
+      "description": "The apiProfile to use for all resources in the template."
+    },
+    "$schema": {
+      "type": "string",
+      "description": "JSON schema reference"
+    },
+    "contentVersion": {
+      "type": "string",
+      "pattern": "(^[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+$)",
+      "description": "A 4 number format for the version number of this template file. For example, 1.0.0.0"
+    },
+    "variables": {
+      "type": "object",
+      "description": "Variable definitions"
+    },
+    "parameters": {
+      "type": "object",
+      "description": "Input parameter definitions",
+      "additionalProperties": {
+        "$ref": "#/definitions/parameter"
+      }
+    },
+    "functions": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/functionNamespace"
+      },
+      "description": "User defined functions"
+    },
+    "outputs": {
+      "type": "object",
+      "description": "Output parameter definitions",
+      "additionalProperties": {
+        "$ref": "#/definitions/output"
+      }
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "contentVersion",
+    "resources"
+  ],
+  "definitions": {
+    "2019-03-01-hybrid-profile-resource": {
+      "oneOf": [
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults"
+        },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults_secrets"
```

Comment:
Please ensure we have a test that will break if one of these referenced resources is changed (for example if `https://schema.management.azure.com/schemas/2016-10-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults_secrets` is deleted or renamed).

Please also ensure we have a clear list of instructions/script that can be run to unblock RPs if this test fails.